students/cfarris/lessons/lesson03/assignment/test_basic_operations.py

```diff
@@ -1,11 +1,16 @@
 #!/usr/bin/env Python3
+
+'''
+This pytest suite tests basic functionality of basic_operations.py
+'''
+
 import pytest
-from basic_operations import add_customer, search_customer, delete_customer
+from basic_operations import add_customer, search_customer, delete_customer, update_customer_credit, list_active_customers
```

Comment:
This one you probably do want to split into two lines. Or you can do like I did and just import basic_operations as bs or bo, and use them bo.add_customer, etc. Which is kind of fun.